### ISSUE TITLE
Engine hangs with .pvtu files.

### ISSUE THREAD

**Issue by aowen87** [aowen87/FOO#261]
David Zwick reported this on users list.
He has time-varying .pvtu files. When opened with visit in parallel, the engine hangs.
I looked at log files and it appears the processors are waiting in different mpi gather calls:
proc 0 in Engine::GatherData and proc 1 in avtLocalStructuredDomainBoundaryList::GlobalGather

Don't know why we would be using a 'Structured' domain boundary list on unstructured data....

I've attached his sample files.  Open the part000*.pvtu virtual database that visit groups.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 05/11/2018 01:02 pm
Original update: 05/29/2018 06:54 pm
Ticket number: 3102
